test/service.test.ts

```diff
@@ -24,6 +24,31 @@ test('Must be configured with at least one port', () => {
 
 });
 
+test('Can provide cluster IP', () => {
```

Comment:
I added this unit test to validate that the output spec does indeed have "clusterIP" and not "clusterIp" since that is what kubernetes expects.

Reply:
👍